__tests__/app.test.js

```diff
@@ -104,8 +109,63 @@ describe('/api/articles/:article_id', () => {
                 });
         });
     })
+    describe('PATCH update an article by article_id', () => {
+        test('PATCH 200: Should respond with 200 and show the updated article', () => {
+            const updateToArticle = {
+                inc_votes:2
+            }
+            return request(app)
+                .patch('/api/articles/3')
+                .send(updateToArticle)
+                .expect(200)
+                .then(({ body }) => {
+                    const { article } = body
+                    expect(article.votes).toBe(2)
+                })
```

Comment:
It might be more sturdy to test against the entire response coming back from this request here, currently we could just send back this one votes property and pass the test when actually we will want the full response to be there and so should test for it.